### PR TITLE
[HK] Remove trailing whitespaces and several other fixes

### PR DESCRIPTION
**Pull request by sschulz92**
related to styling

### CODE REVIEW

File: plugins/org.locationtech.udig.catalog.ui/src/org/locationtech/udig/catalog/service/database/DatabaseServiceDialect.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
- */
+*    uDig - User Friendly Desktop Internet GIS client
+*    http://udig.refractions.net
+*    (C) 2012, Refractions Research Inc.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+* License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+*/
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.catalog.ui/src/org/locationtech/udig/catalog/service/database/DatabaseServiceDialect.java:10]: here the like define in codetemplates.xml we should keep the space at the beginning

File: plugins/org.locationtech.udig.project.ui.tests/src/org/locationtech/udig/project/ui/internal/tool/display/ToolManagerTest.java [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
- */
+*    uDig - User Friendly Desktop Internet GIS client
+*    http://udig.refractions.net
+*    (C) 2012, Refractions Research Inc.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+* License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+*/
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.project.ui.tests/src/org/locationtech/udig/project/ui/internal/tool/display/ToolManagerTest.java:10]: same here, class comments should not be changed

File: plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ReplaceableSelectionProvider.java [hunk 3]
```diff
@@ -19,65 +19,76 @@
...
  * Since a MapEditor can have a different selection provider depending on the tool that is selected, this class
- * is needed to manage the changing selection providers.  
- * 
+ * is needed to manage the changing selection providers.
+ *
  * @author Jesse
  * @since 1.1.0
  */
-class ReplaceableSelectionProvider implements ISelectionProvider {
+public class ReplaceableSelectionProvider implements ISelectionProvider {
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ReplaceableSelectionProvider.java:27]: should it be visible from outside of the package?

File: plugins/org.locationtech.udig.tutorials.preferredzoomlevels/src/org/locationtech/udig/tutorials/preferredzoomlevels/View.java [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
- * License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
- */
+* http://udig.refractions.net
+* (C) 2011, Refractions Research Inc.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+* License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+*/
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.tutorials.preferredzoomlevels/src/org/locationtech/udig/tutorials/preferredzoomlevels/View.java:9]: same here, class comments should not be changed

File: plugins/org.locationtech.udig.tutorials.tool-view/src/org/locationtech/udig/tutorials/toolview/View.java [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
- * License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
- */
+* http://udig.refractions.net
+* (C) 2011, Refractions Research Inc.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+* License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+*/
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.tutorials.tool-view/src/org/locationtech/udig/tutorials/toolview/View.java:9]: same here, class comments should not be changed

File: plugins/org.locationtech.udig.ui/src/org/locationtech/udig/internal/ui/UiPlugin.java [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
- */
+*    uDig - User Friendly Desktop Internet GIS client
+*    http://udig.refractions.net
+*    (C) 2012, Refractions Research Inc.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+* License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+*/
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.ui/src/org/locationtech/udig/internal/ui/UiPlugin.java:10]: same here, class comments should not be changed

**Review by fgdrf — CHANGES_REQUESTED** [review]

**Comment by sschulz92** [comment]
I just uploaded a new commit to my PR @fgdrf :)

You mentioned the "codetemplates.xml" file which I also updated. A "normal" comment starts with `/**` which was not the case. What do you think?

File: plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ReplaceableSelectionProvider.java (hunk 3 above)
Comment (sschulz92) [reply plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ReplaceableSelectionProvider.java:27]: I am using this class in a self-implemented ViewPart therefore I need to access it :)

**Pull request merged**